doc/changes/latest.rst

```diff
@@ -16,3 +16,5 @@ Enhancements
 
 Changes
 -------
+
+- :bdg-info:`Plotting` Change the default map to be ``"RdBu_r"`` or ``"gray"`` for most plotting functions (:gh:`4807` by `Rémi Gau`_).
```

Comment:
Looks like we have used inferno in more places than RdBu_r. I guess this that's because they look similar?

Reply:
I think this impression comes from the fact that I have 'fixed' / 'updated' the examples to use a proper sequential color map.

There a were a lot of examples that used the default diverging colormap when a sequential colormap was actually better suited.

Will update the changelog.